02-14-2021-21-12-38[ss][EXCEPTION]: Traceback (most recent call last):
  File "/home/spotify/projects/zeus/src/services/driversadapter/chrome.py", line 133, in getNewInstance
    driver = WebDriver(options=options, desired_capabilities=desired_capabilities)
  File "/home/spotify/.local/lib/python3.9/site-packages/selenium/webdriver/chrome/webdriver.py", line 73, in __init__
    self.service.start()
  File "/home/spotify/.local/lib/python3.9/site-packages/selenium/webdriver/common/service.py", line 102, in start
    time.sleep(1)
KeyboardInterrupt

